srv/PutInformJob.srv

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: 2024, Yaskawa America, Inc.
+# SPDX-FileCopyrightText: 2024, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# The request does not provide a 'name' field as MotoROS2 will use the job name
+# specified in the job header (ie: "//NAME <JOB_NAME>")
```

Comment:
How sure are you about that?
I know that you'll get an error on the pedant if you try to load from the USB drive and the filename doesn't match the header name.